src/contracts/pods/EigenPod.sol

```diff
@@ -428,6 +432,37 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         _sendETH(recipient, amountWei);
     }
 
+
+    /*******************************************************************************
+                    EXTERNAL FUNCTIONS CALLABLE BY PERMISSIONED SERVICES
+    *******************************************************************************/
+
+    /// @notice Called by the EigenPodManager to fulfill a partial withdrawal proof request
+    function fulfillPartialWithdrawalProofRequest(
+        IEigenPodManager.WithdrawalCallbackInfo calldata withdrawalCallbackInfo,
+        address feeRecipient
+    ) external onlyEigenPodManager  {
+        uint256 provenPartialWithdrawalSumWei = withdrawalCallbackInfo.provenPartialWithdrawalSumWei;
+
+        require(withdrawalCallbackInfo.startTimestamp < withdrawalCallbackInfo.endTimestamp, "EigenPod.fulfillPartialWithdrawalProofRequest: startTimestamp must precede endTimestamp");
```

Comment:
Is `<=` valid?

Reply:
We're doing  non inclusice end timestamp, ie, (start, end] so we want "<"